reference.rst

```diff
@@ -8,8 +8,10 @@ Reference
    :maxdepth: 2
    :titlesonly:
    
-   reference/commands
-   reference/conanfile
-   reference/conanfile_txt
-   reference/tools
-   reference/extensions
+    reference/commands
+    reference/conanfile
+    reference/conanfile_txt
+    reference/tools
+    reference/profiles
+    reference/configuration_files
```

Comment:
I wonder if profiles should be considered configuration files

Reply:
Yeah, let me refactor that. I was wondering the same...